02 - Community Expansions/02.04 Auxiliary Tools by Category/iOS Shortcuts.md

```diff
@@ -2,15 +2,17 @@
 aliases: 
 - 
 tags:
-- seedling
+- evergreen
 publish: true
 ---
 
 # iOS Shortcuts
 
 %% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
 
-#placeholder/description 
+iOS/iPadOS/MacOS [Shortcuts]([](https://support.apple.com/guide/shortcuts/welcome/ios)) for use in conjunction with Apple's [Shortcuts]([https://apps.apple.com/us/app/shortcuts/id915249334](https://apps.apple.com/us/app/shortcuts/id915249334)) app. 
```

Comment:
The text in the links here looks very odd. Almost like you tried pasting a URL, and some Obsidian plugin mangled the URL:
```text
[Shortcuts]([](https://support.apple.com/guide/shortcuts/welcome/ios))
[Shortcuts]([https://apps.apple.com/us/app/shortcuts/id915249334](https://apps.apple.com/us/app/shortcuts/id915249334))
```

Reply:
It would most likely have been the "Auto link title" plugin or similar... It happens to me too.

Reply:
So strange. This isn’t due to a plug-in; this is just what happens when I paste a link from Safari into Obsidian on my iPad.